qiskit_experiments/data_processing/data_action.py

```diff
@@ -81,16 +102,42 @@ def __repr__(self):
 class TrainableDataAction(DataAction):
     """A base class for data actions that need training."""
 
+    def __init__(self, validate: bool = True):
```

Comment:
This looks fine. Just to recap: you serialize standard nodes with the init args and kwargs. `TrainableDataAction` have the added complexity that you need to serialize the parameters of the trained node correct? This sounds reasonable. Can you add some docstrings to explain to potential developers that trainable nodes need to use the `self._parameters` to be properly serializable?

Reply:
That is correct. added in https://github.com/Qiskit/qiskit-experiments/pull/639/commits/1c0b99d7f26afbe79bec1e1cb2d0d19320087dc0